sdk/rust-examples/fd_create/src/main.rs

```diff
@@ -0,0 +1,69 @@
+// This test program is not ready to be released!
```

Comment:
Can you add the licensing/copyright header?

Reply:
Done.